webpack.config.js

```diff
@@ -7,10 +7,13 @@ const outputName = 'app-webpack';
 
 module.exports = ({ env = 'dev', plugins = [] } = {}) => ({
     devtool: env === 'dev' ? 'inline-source-map' : 'source-map',
-    entry: path.join(__dirname, 'static', 'src', 'javascripts', 'boot-webpack.js'),
+    entry: {
+        [outputName]: path.join(__dirname, 'static', 'src', 'javascripts', 'boot-webpack.js'),
+        admin: path.join(__dirname, 'static', 'src', 'javascripts-legacy', 'bootstraps', 'admin.js'),
+    },
     output: {
         path: path.join(__dirname, 'static', 'target', 'javascripts'),
-        filename: `${env === 'dev' ? '' : '[chunkhash]/'}${outputName}.js`,
+        filename: `${env === 'dev' ? '' : '[chunkhash]/'}[name].js`,
         chunkFilename: `${env === 'dev' ? '' : '[chunkhash]/'}${outputName}.chunk-[id].js`,
```

Comment:
Yeah, I initially tried that, but it doesn't work out quite like that! 

`[name]` in the context of `chunkFilename` refers to the name given to the chunk, an optional [third parameter passed to `require.ensure`](https://webpack.js.org/guides/code-splitting-require/#components/sidebar/sidebar.jsx). If no name is passed, [`[name]` becomes the `[id]`](https://webpack.js.org/configuration/output/#output-chunkfilename). So we end up with chunks called `0.chunk-0.js`, and so on.

I thought: why doesn't `[name]` just take the name of the parent entry chunk? It's because a chunk could have multiple entry chunks.

Let me know if you think of a preferable way of doing this